bin/configs/kotlin-allOff-discriminator-kotlinx-serialization.yaml

```diff
@@ -0,0 +1,10 @@
+generatorName: kotlin
+outputDir: samples/client/petstore/kotlin-allOff-discriminator-kotlinx-serialization
```

Comment:
thanks for the PR. what about adding the new folder to the github workflow so that it will be tested moving forward?

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-kotlin-client.yaml